css/index.css

```diff
@@ -1,3 +1,314 @@
+	border-bottom: 1px solid black;
+	display: flex;
+	justify-content: space-between;
+	padding-bottom: 5%;
+	margin-top: 5%;
+	margin-bottom: 5%;
+}
+
+#main-banner section {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	margin-left: 8%;
+}
+
+#main-banner section h1 {
+	font-size: 6.5em;
+	text-align: center;
+}
+
+#main-banner section button {
+	background-color: white;
+	border: .5px solid black;
+	padding: 0 10%;
+	font-size: 1em;
+	font-family: 'Titillium Web';
+}
+
+#main-banner img {
+	width: 40%;
+	height: 40%;
+}
+
+#row-1 {
+	display: flex;
+	margin-bottom: 5%;
+}
+
+.row-1-content h2 {
+	font-size: 1em;
+}
+
+.row-1-content p {
+	font-size: 1em;
+}
+
+.middle-img {
+	width: 100%;
+	margin-bottom: 5%;
+}
+
+#row-2 {
+	border-bottom: 1px solid black;
+	display: flex;
+	padding-bottom: 5%;
+	margin-bottom: 3.5%;
+}
+
+.row-2-content h2 {
+	font-size: 1em;
+}
+
+.row-2-content p {
+	font-size: 1em;
+}
+
+footer h3, address, p {
+	font-size: 1em;
+}
+
+footer p {
+	text-align: center;
+	margin-top: 5%;
+	padding-bottom: 10%;
+}
+}
+
+/* Tablet Code Version */
+
+@media (max-width: 800px) {
+
+  body {
+    margin: 0 1%;
+  }
+
+  header {
+    flex-direction: column-reverse;
+    align-items: center;
+  }
+
+  .logo {
+    width:25%;
+    padding-top: 10px;
+    padding-bottom: 0;
+  }
+
+  header nav{
+    width: 100%;
+    font-size: 13px;
+    padding-top: 25px;
+    padding-bottom: 30px;
+    margin: 0 5px 5px 0;
+  }
+
+  #header-image {
+    display: none;
+  }
+
+  #main-banner {
+    justify-content: center;
+  }
+
+  #main-banner section h1 {
+    font-size: 60px;
+    }
+
+  #main-banner section {
+    width: 100%;
+    margin: 0;
+    margin-bottom: 20px;
+  }
+
+  #main-banner section button {
+    padding-left: 35px;
+    padding-right: 35px;
+  }
+
+  #main-banner {
+    border-bottom: 1px solid black;
+  }
+
+  #row-1, #row-2 {
+    font-size: 13px;
+    font-weight: bold;
+  }
+
+  .row-1-content, .row-2-content {
+    padding: 8px;
+  }
+
+  #row-2 {
+    border-bottom: 1px solid black;
+  }
+
+  footer p {
+    font-size: 13px;
+  }
+
+}
+
+/* Mobile Code Version */
+
+@media (max-width: 500px) {
+
+  body {
+    margin: 0;
+  }
+
+  .logo {
+    width: 35%;
+  }
+
+   header nav {
+    display: flex;
+    flex-wrap: wrap;
+    font-size: 15px;
+  }
+
+  header nav a {
+    width: 500px;
+    text-align: center;
+    padding: 10px 0;
+    border-top: 1px solid grey;
+    border-bottom: .5px solid grey;
+  }
+
+  header nav a:first-child {
+    border-top:1.5px solid grey;
+  }
+
+  header nav a:last-child {
+    border-bottom: 1.5px solid grey;
+  }
+
+  #row-1, #row-2 {
+    flex-wrap: wrap;
+  }
+
+  .row-1-content, .row-2-content {
+    width: 100%;
+    padding-top: 10px;
+    padding-bottom: 10px;
+  }
+
+  #row-2 {
+	border-bottom: 1px solid black;
+  }
+
+  #main-banner {
+    border-bottom: 1px solid black;
+  }
+}
+
+/* -- - Stop - -- */
+
+/* -- - Mistakes - -- */
```

Comment:
Have you thought about re arranging your HTML to have the logo be on top? Also, remember you can also change the order of your elements with flex.